include/dav1d/headers.rs

```diff
@@ -193,13 +193,38 @@ pub const DAV1D_RESTORATION_WIENER: Dav1dRestorationType =
 pub const DAV1D_RESTORATION_SGRPROJ: Dav1dRestorationType =
     Rav1dRestorationType::SgrProj as Dav1dRestorationType;
 
-#[derive(Debug, Clone, Copy, PartialEq, Eq, FromRepr, Default)]
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, FromRepr, Default)]
+#[repr(u8)]
```

Comment:
We shouldn't need this.  It'll choose the smallest size anyways and specifying a repr disables niche optimizations I believe.